### PR TITLE
NMS-10155: Document Newts settings

### PR DESCRIPTION
**Pull request by emarsh1995**
- Update `deployment/time-series-storage/newts/newts.adoc`

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-10155

### CODE REVIEW

File: docs/modules/deployment/pages/time-series-storage/newts/newts.adoc
Comment (Bonrob2) [inline docs/modules/deployment/pages/time-series-storage/newts/newts.adoc:24]:
```suggestion
<2> Set time series strategy to use Newts.
```
Thinking it refers to the feature here.

File: docs/modules/deployment/pages/time-series-storage/newts/newts.adoc
Comment (Bonrob2) [inline docs/modules/deployment/pages/time-series-storage/newts/newts.adoc:25]:
```suggestion
If you are enabling the xref:time-series-storage/timeseries/time-series-storage.adoc#ga-dual-write-newts[dual write plugin] on an existing installation and want to keep historical metrics, do not set the `org.opennms.timeseries.strategy` to `newts` until you have written enough data to expire any collected data prior to enabling the dual write feature.
```
The last part of the sentence doesn't quite parse. Is it to expire the data that was collected before enabling the dual write feature, or don't do this until you make sure you've expired the data, then enable the dual write feature?

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
A few minor changes. Thanks!

File: docs/modules/deployment/pages/time-series-storage/newts/newts.adoc
Comment (emarsh1995) [reply docs/modules/deployment/pages/time-series-storage/newts/newts.adoc:25]: I think it's the latter, but I'll defer to @jeffgdotorg for confirmation.

Comment (Bonrob2) [reply docs/modules/deployment/pages/time-series-storage/newts/newts.adoc:25]: Everything looks good to me. Will approve once we hear back from Jeff about the above comment.

Comment (jeffgdotorg) [reply docs/modules/deployment/pages/time-series-storage/newts/newts.adoc:25]: I'm reasonably confident it's meant to be the latter.

**Review by jeffgdotorg — CHANGES_REQUESTED** [review]
See comment, that one sentence does need a rewrite.

File: docs/modules/deployment/pages/time-series-storage/newts/newts.adoc
Comment (emarsh1995) [reply docs/modules/deployment/pages/time-series-storage/newts/newts.adoc:25]: @jeffgdotorg, does this work better? Is there any more specific information that you think would make the sentence parse more naturally?

```suggestion
If you are enabling the xref:time-series-storage/timeseries/time-series-storage.adoc#ga-dual-write-newts[dual write plugin] on an existing {page-component-title} installation and you want to keep historical metrics, make sure that the written data has expired before you set `org.opennms.timeseries.strategy` to `newts`.
```

Comment (Bonrob2) [reply docs/modules/deployment/pages/time-series-storage/newts/newts.adoc:25]: This works for me!

**Pull request merged**
